apps/aecore/lib/aecore/chain/block_validation.ex

```diff
@@ -123,13 +123,13 @@ defmodule Aecore.Chain.BlockValidation do
     end)
   end
 
-  @spec calculate_txs_hash(list(SignedTx.t())) :: binary()
-  def calculate_txs_hash(txs) when txs == [] do
+  @spec calculate_root_hash(list(SignedTx.t())) :: binary()
+  def calculate_root_hash(txs) when txs == [] do
     <<0::256>>
   end
 
-  @spec calculate_txs_hash(list(SignedTx.t())) :: binary()
-  def calculate_txs_hash(txs) do
+  @spec calculate_root_hash(list(SignedTx.t())) :: binary()
+  def calculate_root_hash(txs) do
```

Comment:
this function does `calculate_txs_hash`

Reply:
the txs_hash is the merkle roota hash of the transactions tree included within one block